server/routes/api/campaigns.js

```diff
@@ -251,7 +251,40 @@ router.delete(
       });
     } catch (error) {
       console.log(error);
-      res.json({ error: "Error deleting prospects" });
+      res.status(500).json({ error: "Error deleting prospects" });
+    }
+  }
+);
+
+// @route POST /api/campaigns/:campaignId/steps/:stepId/sendEmails
+// @desc Send email in step to all prospects in that step
+// @access Authenticated Users
+router.post(
+  "/:campaignId/steps/:stepId/sendEmails",
+  passport.authenticate("jwt", { session: false }),
+  async (req, res) => {
+    try {
+      const { campaignId, stepId } = req.params;
+      const userId = req.user.id;
+
+      const campaign = await Campaign.findOne({
+        _id: campaignId,
+        ownedBy: userId
+      });
+
+      if (!campaign)
+        throw new Error(
+          `Campaign ${campaignId} owned by ${userId} cannot be found`
+        );
+
+      await sendEmailsQueue(stepId, req.user.id, req.user.gmailToken);
```

Comment:
instead of call this function, I'd want to call a function that just enqueues one job, that's why earlier I mentioned you should have the Queue created outside of this function sendEmailsQueue

Reply:
Got it. I've changed it so we now create the queue in `controllers/queues/index.js` once and import the queue and add a job to it here.